README.md

```diff
@@ -1 +1,45 @@
-# newman-reporter-html
\ No newline at end of file
+# newman-reporter-html
+Advanced HTML reporter for [newman](https://github.com/postmanlabs/newman).
+
+## Install
```

Comment:
Need a paragraph before this to explain that this is a reporter for Newman that provides HTML output and that it needs to be used in conjunction. And that when installed, Newman CLI will recognise the following options.